volontulo/templates/common/col1.html

```diff
@@ -1,13 +1,21 @@
 {% extends "common/base.html" %}
 
 {% block layout %}
-
-    <div class="row">
-        <div class="col-xs-12">
-        {% block content %}
-        {% endblock %}
+<section class="content-heading">
```

Comment:
What about empty content-heading block? If block would be empty then empty and unneeded section.content-heading>div.container would be displayed.